packages/governance/test/unitTests/test-buildParamManager.js

```diff
@@ -331,6 +331,13 @@ test('JSON blobs', async t => {
     Blob: '{ "authors": "Crockford,Morningstar" }',
   });
   t.is(paramManager.getBlob(), '{ "authors": "Crockford,Morningstar" }');
+
+  await paramManager.updateParams({
+    // trailing whitespace is preserved
+    Blob: '{ "authors": "Crockford,Morningstar" }    ',
+  });
+  t.is(paramManager.getBlob(), '{ "authors": "Crockford,Morningstar" }    ');
```

Comment:
nit: better to make this part of the test output.
```suggestion
    Blob: '{ "authors": "Crockford,Morningstar" }    ',
  });
  t.is(paramManager.getBlob(), '{ "authors": "Crockford,Morningstar" }    ', 'extraneous whitespace must be preserved);
```